code/game/atoms.dm

```diff
@@ -950,14 +950,6 @@
 	SEND_SIGNAL(src, COMSIG_ATOM_RCD_ACT, user, the_rcd, passed_mode)
 	return FALSE
 
-/**
-  * Respond to our atom being teleported
-  *
-  * Default behaviour is to send COMSIG_ATOM_TELEPORT_ACT
-  */
-/atom/proc/teleport_act()
-	SEND_SIGNAL(src,COMSIG_ATOM_TELEPORT_ACT)
-
```

Comment:
Why remove this proc? This forces any atoms that want to have a teleport_act override to register to signals instead of just overriding the proc.
While this is fine for components, and I would rather things be components as it works better for systemic games like SS13 it forces atoms to register signals instead of overriding this which just adds unnecessary runs through the DCS (although thats only particularly bad for hot paths).
Ultimately there is no reason to remove this as it only makes having a specific typepath override the behaviour more impractical.

Reply:
`COMSIG_ATOM_TELEPORT_ACT` has 1 singular use case, which is the zamscode `/mob/living/simple_animal/hostile/breadloaf`, I thought that removing this from *literally every atom* would be better as this only allows for things to react to be teleported, all other reaction code for teleportations like BoH is handled within do_teleport()